FreeAPS/Sources/Shortcuts/Override/overridePresetEntity.swift

```diff
@@ -0,0 +1,33 @@
+import AppIntents
+import Foundation
+import Intents
+import Swinject
+
+@available(iOS 16.0, *) struct overridePreset: AppEntity, Identifiable {
```

Comment:
Please capitalize the `struct` names here and the file name to conform to how other `struct`s in the project are named to far, i.e. 
- `overridePresetEntity.swift` -> `OverridePresetEntity.swift`
- `overridePreset` -> `OverridePreset`
- `overridePresetsQuery` -> `OverridePresetsQuery`

Reply:
@dnzxy Xcode was misbehaving, fixed now